scripts/mock-blogdata.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "imageUrl": "https://www.icicidirect.com/images/Blog-202306051107590806957.jpg",
```

Comment:
Wondering if we should use sample images placed in Sharepoint and use the media-query urls here for mock data, some of these URLs would be invalid over time.

Reply:
@vivesing  currently this is mock data only, when Api response comes in url would be generated on based on api response. IMO we can keep it as it for now and whenever the response is ready we csn take it from there

Reply:
My only concern is some of the images never show up, this might affect the scores in general and affects the look and feel once we share it for trial. It would be better if we could use a same image on all tiles that has been published from our repo. (minor concern tough)
![image](https://github.com/aemsites/icicidirect/assets/38683124/89f642ff-4d2a-435f-b362-7bd0561e8d46)